### ISSUE TITLE
Lazy load "On Scroll" only work if scroll is possible

### ISSUE THREAD

**Issue by flemmingss** [iamsayan/ultimate-facebook-comments#1]
If your post/page and all its content fits inside your browser windows it is not possible to get the comments to show.
This is a problem, I assume all people that select this option wants comments to show up in these cases.

This screenshot demo the problem:
![demo](https://user-images.githubusercontent.com/25674067/62012654-4fdc8a00-b189-11e9-9003-fb7f18599aff.png)

**Comment by iamsayan** [comment]
Hi,

You can use the Default mode in this case. Thanks!

**Comment by flemmingss** [comment]
Yeah, That is what I use, but this for information so it can be fixed ;)
